scripts/release.sh

```diff
@@ -62,3 +62,5 @@ then
 else
     echo -e "\e[32mVulnerability info is already added for version: $VERSION, nothing to do here.\e[0m"
 fi
+
+npx @alfresco/adf-cli@alpha changelog -o $ROOTDIR/docs/changelog
```

Comment:
we don't store it there, we save it in the release file, maybe we can do something that copy the output in the release file if there is a tag there?